docs/tutorials/ionq/getting_started.ipynb

```diff
@@ -0,0 +1,328 @@
+{
```

Comment:
<p>A few clarity/grammar/spelling nits here: </p><p><code>supports a set of gates</code>  -&gt; <code>supports a limited set of gates</code>  </p><p><code>Circuit build</code>  -&gt; <code>Circuits built</code>  </p><p><code>The follow</code>  -&gt; <code>The following cell</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3758/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>fixed</p>